src/components/molecules/SfOptions/SfOptions.html

```diff
@@ -0,0 +1,27 @@
+<div class="sf-options">
+  <sf-label v-if="label" class="sf-options__title">
+    {{ label }}
+  </sf-label>
+  <div class="sf-options__wrapper">
+    <div
+      v-for="option in options"
+      :key="option.id"
+      class="sf-options__option-container"
+      :class="{ 'sf-options__option-container--selected': isOptionSelected(option.value) }"
+      @click="setActiveValue(option.value)"
+    >
+      <slot>
```

Comment:
Sorry for not being precise enough.

My idea was to have a separate slot for every type of Option. With current implementation if someone is willing to change the way color switch looks like he/she needs to create new markup for the rest too. having it splitted allows more flexibility I believe. If you think full replacement is a needed usa case we can have them wrapped in one general slot (so current one can remain) but separate ones are also needed

Reply:
It was my misunderstanding. I have updated by separating slot as type and binding value. Only one slot is too strict. I hope that it should be more flexible and easier to read now. 😁